py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -263,22 +244,21 @@ def aten_ops_embedding(
         name,
         input=args[1],
         weight=args[0],
-        # args[2] is the padding index, which is useful for training only
-        scale_grad_by_freq=args_bounds_check(args, 3),
-        sparse=args_bounds_check(args, 4),
+        # args[2, 3, 4] are useful for training only
+        padding_idx=args_bounds_check(args, 2, -1),
+        scale_grad_by_freq=args_bounds_check(args, 3, False),
+        sparse=args_bounds_check(args, 4, False),
```

Comment:
If these are training-only parameters (no effect at inference time), it is safe to omit them entirely (ignore those args). If they do have an effect at inference time, but we don't support that effect, then a validator can be used. It seems the validator is removed so I assume it is the former case.